lib/pliny/middleware/rescue_errors.rb

```diff
@@ -8,25 +8,12 @@ def initialize(app, options = {})
     def call(env)
       @app.call(env)
     rescue Pliny::Errors::Error => e
-      # blank out this field so that the error is not picked up by Rollbar
-      env["sinatra.error"] = nil
-
       Pliny::Errors::Error.render(e)
     rescue Exception => e
```

Comment:
Rescuing `Exception` and not re-raising it causes interrupts and sigkills to be ignored. Rescuing `StandardError` might make more sense here, or alternatively, re-raising if the error is not a `StandardError`.